scheduler/scheduler.go

```diff
@@ -322,14 +332,8 @@ func executeJob(j gaia.Job, pS Plugin, triggerSave chan gaia.Job) {
 
 	// Execute job
 	if err := pS.Execute(&j); err != nil {
-		// TODO: Show it to user
 		gaia.Cfg.Logger.Debug("error during job execution", "error", err.Error(), "job", j)
-		j.Status = gaia.JobFailed
```

Comment:
I'm pretty sure you want to set the job status to failed here? How come you deleted all this code? If you don't have to return any more.

Reply:
I'm passing the job pointer to `pS.Execute` and there is set the status now: https://github.com/gaia-pipeline/gaia/pull/101/files#diff-daed06840ae2242793f6bca24ba9b8bdR185